android/src/main/java/com/reactcommunity/rndatetimepicker/MaterialRangePickerModule.kt

```diff
@@ -0,0 +1,45 @@
+package com.reactcommunity.rndatetimepicker
+
+import androidx.fragment.app.FragmentActivity
+import com.facebook.react.bridge.Promise
+import com.facebook.react.bridge.ReactApplicationContext
+import com.facebook.react.bridge.ReadableMap
+import com.facebook.react.bridge.UiThreadUtil
+import com.reactcommunity.rndatetimepicker.Common.createDatePickerArguments
+import com.reactcommunity.rndatetimepicker.Common.createRangePickerArguments
+import com.reactcommunity.rndatetimepicker.Common.dismissDialog
+
+class MaterialRangePickerModule(reactContext: ReactApplicationContext): NativeModuleMaterialRangePickerSpec(reactContext) {
+  companion object {
+    const val NAME = "RNCMaterialRangePicker"
```

Comment:
If I'm not mistaken, `NativeModuleMaterialRangePickerSpec` generated by codegen already should contain the name, and so specifying it here should not be needed. also `override fun getName` should be in the generated code.
if not, it's fine